pkg/controller/serviceaccount/serviceaccounts_controller.go

```diff
@@ -208,6 +208,9 @@ func (c *ServiceAccountsController) syncNamespace(key string) error {
 		case err != nil:
 			return err
 		}
+		// inherit Labels from namespace
+		sa.Labels = ns.Labels
```

Comment:
I wouldn't expect this.  Why would the service account get the labels of the namespace?